spiped/main.c

```diff
@@ -27,6 +28,12 @@ usage(void)
 	exit(1);
 }
 
+static void
+signal_handler(int signo)
+{
+	exit(0);
```

Comment:
After the blank line which separates variables from code.  Before _exit(0) since otherwise it would be dead code.

So the function should look like this:

    {
        [blank line]
        (void)signo; /* UNUSED */
        _exit(0);
    }